### PR TITLE
[KIECLOUD-234] Field validations on change of their values

### PR DESCRIPTION
**Pull request by swati-kale**
Signed-off-by: Swati Kale <swkale@redhat.com>

### CODE REVIEW

File: frontend/src/component/operator-wizard/page-component/Page.js [hunk 1]
```diff
@@ -97,12 +97,14 @@ export default class Page extends Component {
   render() {
     return (
       <Form id={"form-page-" + this.props.pageNumber}>
-        <Title headingLevel="h1" size="3xl">
-          {this.props.pageDef.label}
-        </Title>
-        {this.state.elements.map(element => {
-          return element.getJsx();
-        })}
+        <div key={"page" + this.props.pageNumber}>
```
Comment (ruromero) [inline frontend/src/component/operator-wizard/page-component/Page.js]: Why adding a div here? Why not set the key in the Form component?

Comment (swati-kale) [reply frontend/src/component/operator-wizard/page-component/Page.js]: removed the div

**Comment by swati-kale** [comment]
https://issues.jboss.org/browse/KIECLOUD-234

**Comment by ruromero** [comment]
Closing for #73

**Pull request closed without merging**